app/Http/Requests/Tasks/UpdateRequest.php

```diff
@@ -25,11 +25,21 @@ public function authorize()
      */
     public function rules()
     {
-        return [
-            'name'        => 'required|max:60',
-            'description' => 'nullable|max:255',
-            'progress'    => 'required|numeric|max:100',
-            'job_id'      => 'required|numeric|exists:jobs,id',
+        $rules = [
+            'name'     => 'required|max:60',
+            'progress' => 'required|numeric|max:100',
+            'job_id'   => 'required|numeric|exists:jobs,id',
         ];
+
+        //Allow for flexibility instead of optionless hard-coded value for "description". This is
+        //achieved using environmental variable.
+        //A value of zero (0) will mean "no limit"
+
+        $char_len_task_description = intval(env('CHAR_LEN_TASK_DESCRIPTION', 255));
```

Comment:
Change this to **DESCRIPTION_CHAR_LEN**, too. And change variable name to **camelCase**.